src/app/shared/shared.module.ts

```diff
@@ -1,19 +1,39 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
 
-import { AuthGuard, DeactivateGuard, UnauthGuard, ImageModule, SpinnerModule } from 'ngx-prx-styleguide';
+import { AuthGuard, DeactivateGuard, UnauthGuard,
+  FancyFormModule, HeroModule, ImageModule, ModalModule,
+  SpinnerModule, TabModule } from 'ngx-prx-styleguide';
+
+import { TimeAgoPipe } from './date';
+import { TimeFromNowPipe } from './date';
 
 @NgModule({
-  declarations: [],
+  declarations: [
+    TimeAgoPipe,
+    TimeFromNowPipe
+  ],
   exports: [
     CommonModule,
+    FancyFormModule,
     ImageModule,
-    SpinnerModule
+    HeroModule,
+    ModalModule,
+    SpinnerModule,
+    TabModule,
```

Comment:
Having the TabModule as part of the SharedModule is what caused those bugs in Publish where the TabService was returning story models for the series because there was only one instance of the service. If there are other modules planned with tabs, you'll probably want to give the campaign module it's own TabService by including the TabModule there directly rather than through the SharedModule. Otherwise, I do feel like having these common things shared through the SharedModule is the more convenient way to go.

Reply:
I don't believe that there will end up being other modules with tabs, but just in cases, I went ahead and moved it to the CampaignModule -- can revert if you prefer